src/project/qml/MuseScore/Project/internal/PropertyItem.qml

```diff
@@ -103,13 +103,23 @@ RowLayout {
         }
     }
 
+    StyledTextLabel {
+        Layout.fillWidth: root.valueFillWidth
+        Layout.preferredWidth: -1
```

Comment:
This -1 looks odd. Could you please explain the purpose of it?

Reply:
This might sound bad, but I don't remember why I put that :). I think I was working on a different solution which needed something like that. I'll remove that